task/task.json

```diff
@@ -149,6 +149,15 @@
       "helpMarkDown": "If checked, all the work items that linked to the commits will be linked also to the PullRequest.",
       "visibleRule": "repoType=Azure DevOps"
     },
+    {
+      "name": "passPullRequestIdBackToADO",
+      "type": "boolean",
+      "label": "Pass Pull Request ID back to Azure DevOps as a variable",
+      "defaultValue": false,
+      "required": false,
+      "helpMarkDown": "If checked, the Pull Request ID will be passed back to Azure DevOps for use in other pipeline tasks.  The variable can be referenced as '$(pullRequestId)'.  Note this feature is disabled if multiple target branches are specified with a wildcard (e.g. feature/*)",
+      "visibleRule": "repoType=Azure DevOps"
```

Comment:
Hi,
I see also in GitHub PR creation you added the `setvariable...` so you can remove the `"visibleRule": "repoType=Azure DevOps"`.

Reply:
I agree- there's no reason to have that feature available to only Azure DevOps repos.  I've removed the visibility rule.